sqlobject/src/main/java/org/jdbi/v3/sqlobject/customizers/RegisterObjectArgumentFactory.java

```diff
@@ -42,7 +42,7 @@
      * The argument type(s) which are compatible with {@link java.sql.PreparedStatement#setObject(int, Object)}.
      * @return the argument types
      */
-    Class<? extends Class>[] value();
+    Class<? extends Class<?>>[] value();
```

Comment:
This annotation is untested.

Since `Class` is final, the only possible value that can be given to this annotation property is `Class.class`.